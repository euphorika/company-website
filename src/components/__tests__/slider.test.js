import React from "react"
import renderer from "react-test-renderer"

import Slider from "../slider"

describe("Slider", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Slider>
          <img src="https://example.com/img1.jpg" />
          <img src="https://example.com/img2.jpg" />
        </Slider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
