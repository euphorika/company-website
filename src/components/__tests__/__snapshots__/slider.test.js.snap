// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider renders correctly 1`] = `
<div
  className="sliderContainer"
>
  Slider
</div>
`;
